.gitignore

```diff
@@ -16,6 +16,15 @@ autom4te.cache
 *.includes
 CMakeLists.txt.user
 
+
+# Code::Blocks project and backup files
```

Comment:
Reverted in https://github.com/behnle/psi4-1/commit/eaaa1d53ce6905cf37d5a9dc90f18f7c0d55b56a
As there were already entries related to QtCreator in .gitignore, i thought it might be appropriate to add stuff related to CodeBlocks.

Reply:
in  https://github.com/behnle/psi4-1/commit/fb1f631ca8b3a0383a227a659bb6996ff0dc4ca0 i added test cases for 
- closed shell and open shell (UHF) canonical REMP energies
- closed shell and open shell (UHF) DF-REMP energies
- closed shell and open shell (UHF) CD-REMP energies
- closed shell and open shell (UHF) canonical OREMP energies and gradients
- closed shell and open shell (UHF) DF-OREMP energies and gradients
- closed shell and open shell (UHF) CD-OREMP energies
which should give complete test coverage for energies and analytical structure gradients. Tests for analytical electrical properties (oeprop) are still missing.